openshift/templates/knp.yaml

```diff
@@ -95,6 +95,52 @@ objects:
               port: 5432
       policyTypes:
         - Ingress
+  - apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy
+    metadata:
+      name: allow-app-to-db
+      labels:
+        app: ${NAME}${SUFFIX}
+    spec:
+      # Allow the Documize App to talk to the patroni
+      # database cluster on the given port.
+      podSelector:
+        matchLabels:
+          statefulset: ${NAME}-patroni${SUFFIX}
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  deploymentconfig: ${NAME}${SUFFIX}
+          ports:
+            - protocol: TCP
+              port: 5432
+      policyTypes:
+        - Ingress
+  - apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy
+    metadata:
+      name: allow-app-to-conversion
+      labels:
+        app: ${NAME}${SUFFIX}
+    spec:
+      # Allow the Documize App to talk to the Documize Conversion
+      # pods on the given ports.
+      podSelector:
+        matchLabels:
+          deploymentconfig: ${NAME}${SUFFIX}
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  deploymentconfig: ${NAME}-conversion${SUFFIX}
+          ports:
+            - protocol: http
```

Comment:
When i apply them, got error message. It doesn't look like `http` and `https` are supported protocol. If you run `oc explain NetworkPolicy.spec.ingress.ports.protocol` only TCP, UDP, or SCTP are allowed.